presto-main/etc/catalog/sheets.properties

```diff
@@ -0,0 +1,9 @@
+connector.name=sheets
+credentials-path={{ service account credentials file path }}
```

Comment:
maybe hardcode some arbitrary path that deloverper can have locally. Also you can mention in Readme how such file can be generated (if you haven't already).